app/src/main/res/values/strings.xml

```diff
@@ -43,4 +43,4 @@ A difference to existing projects like OpenBCI is that it will not be necessary
     <string name="device_is_not_connected">Device is not connected</string>
     <string name="set_up">Set up</string>
 
-</resources>
+</resources>
```

Comment:
Unnecessary change

Reply:
I didn't change a line there but don't know why it is showing this 😐